src/pages/DefaultOrgSelector/DefaultOrgSelector.jsx

```diff
@@ -64,6 +67,16 @@ function DefaultOrgSelector() {
   const { data: currentUser, isLoading: userIsLoading } = useUser()
   const { mutate: updateDefaultOrg } = useUpdateDefaultOrganization()
 
+  const selectedOrg = orgValue?.org?.username ?? currentUser?.user?.username
```

Comment:
I remember at one point we were using the information from the form itself, has that changed? Did we test changing the form's value and ensuring that's the one we ended up with?

Reply:
this solved the select problem we had, Nick helped me come up with this solution and it resolved the issue with state